operator/config/rbac/role.yaml

```diff
@@ -60,3 +60,19 @@ rules:
   - certificates
   verbs:
   - '*'
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - get
+  - list
+  - create
```

Comment:
why do we need to add these for statefulsets?

Reply:
to create the statefulset of the volume validator

Reply:
and why do we need a statefulset for the volume validator?

Reply:
To deploy it (basing on their repo). i think we might skip it and we should care about having the `VolumePopulator`. But it should verify the source. I didn't test it without having it

Reply:
needs to see if it really needs to scale, if not - there's no need for a statefulset, a regular deployment would be alright

Reply:
dropped